src/Utils/maths.c

```diff
@@ -0,0 +1,30 @@
+#include "math.h"
```

Comment:
```suggestion
#include "maths.h"
```
C'est plutôt `maths` avec un S nan ?